packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -60,6 +60,7 @@ export const PaymentConfirmation = ({
   );
 
   const { l10n } = useLocalization();
+  const downloadLink = productUrl + `/?email=${encodeURIComponent(email)}`;
```

Comment:
The productUrl is a url from the metadata, which could already have query params.  We need to accommodate that.

Reply:
updated to handle either case - query params already present or not.